docs/book/extensions/podcast-index.md

```diff
@@ -0,0 +1,22 @@
+# PodcastIndex Extension
+
+The PodcastIndex Extension adds support for the Podcast Index RSS namespace.
```

Comment:
There is no description of what the extension offers and what advantage it might provide. If the name is changed then the current sentence can be used for every RSS extension.

It would be great if you can add more content for people which do not know anything about "Podcast Index".